tests/testthat/test-shinytest2-teal.R

```diff
@@ -0,0 +1,274 @@
+simple_teal_data <- within(teal_data(), {
+  iris <- iris
+  mtcars <- mtcars
+})
+datanames(simple_teal_data) <- c("iris", "mtcars")
```

Comment:
I'm not a big fan of having objects being reused on multiple tests as they can be modified.

What do you think about changing this to a generator function and moved to the helper?